packages/zap/zap.pacscript

```diff
@@ -0,0 +1,22 @@
+name="zap"
+version="2.2.1"
+build_depends="curl grep jq wget"
```

Comment:
These would be `depends` correct?

Reply:
Yep, dependencies as specified [here](https://github.com/srevinsaju/zap#automated-installation)

Reply:
Yes. All of them are required (though wget is optional).